chrome/background.js

```diff
@@ -99,96 +100,86 @@ function onMessage(request, sender, sendResponse) {
   // spawn a new tab with pre-provided credentials
   if (request.action == "launch") {
     chrome.tabs.create({ url: request.url }, function(tab) {
-      var tabLoaded = false;
```

Comment:
I didn't see a particular use for `tabLoaded`, given that we discard onAuthListener immediately after tab is loaded. Did you have a point for this that I'm missing?

Reply:
That can go. I went through a couple of different versions of the bugfix; that was left over from one of them but is no longer applicable.